app/domain/output_document.rb

```diff
@@ -5,7 +5,9 @@ class OutputDocument
 
   delegate :id, :income_in_retirement, :continue_working, :unsure,
```

Comment:
We only need to delegate attributes that are needed for the output document. That means we can skip the circumstances and possibly pot size and income fields?